README.md

```diff
@@ -281,18 +285,18 @@ stream.on('message', function (msg) {
 
 Emitted each time a status (tweet) comes into the stream.
 
-```javascript
-stream.on('tweet', function (tweet) {
+```js
+stream.on('tweet', tweet => {
   //...
 })
 ```
 
-## event: 'delete'
+## event: 'deconste'
```

Comment:
small typo: should be `delete` here and below

Reply:
this one still needs to be changed back to `delete`, along with the ones below :)